libs/ojsonnet/Value_jsonnet.ml

```diff
@@ -28,7 +28,15 @@ module A = AST_jsonnet
 (* Env *)
 (*****************************************************************************)
 type env = {
-  (* The spec uses a lambda-calculus inspired substitution model, but
+  (* There are currently two implementations of evaluation, one in
+   * Eval_jsonnet, the other in Eval_jsonnet_subst. The former
+   * uses an environment mode, while the later uses lambda calc
+   * substitutions. Currently, the lambda calc version passes more
+   * tests but is inneficient. Keeping both implementations to
+   * possibly mix the two, to get the efficiency benifit of environments
+   * while keeping the simpler super/self implementation given
+   * by substitution model. In the substitution model, we always
```

Comment:
I wonder if we could use Eval_jsonnet.ml even more; like handling self/super with lambda calclus but everything else with the old environment-based evaluator.

Reply:
Yeah, I think I'll try to do that way next. Wanted to get a version of mixing the two models working that sped up the current bottleneck in tests first.